### PR TITLE
Support active_low configuration

### PR DESCRIPTION
**Pull request by tpmanley**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -407,6 +407,35 @@ impl Pin {
...
+    pub fn get_active_low(&self) -> Result<bool> {
+        match self.read_from_device_file("active_low") {
+            Ok(s) => {
+                match s.trim() {
+                    "1" => Ok(true),
+                    "0" => Ok(false),
+                    other => Err(Error::Unexpected(format!("active_low file contents {}", other))),
+                }
+            }
+            Err(e) => Err(::std::convert::From::from(e)),
```
Comment (nastevens) [inline src/lib.rs]: `::std::convert::From::from` is odd... I see that's how it's done in the rest of the file but it can just be done as `e.into()`.

Up to you if you want to fix the other instances in this PR, otherwise I see there are some other warnings to get cleaned up. I could change this along with the other instances in a separate PR.

File: src/lib.rs [hunk 2]
```diff
@@ -407,6 +407,35 @@ impl Pin {
         Ok(())
     }
 
+
```
Comment (nastevens) [inline src/lib.rs]: nit: extra newline

File: CHANGELOG.md [hunk 3]
```diff
@@ -148,6 +154,7 @@
 - Initial version of the library with basic functionality
 - Support for `export`/`unexport`/`get_value`/`set_value`/`set_direction`
 
+[0.5.2]: https://github.com/posborne/rust-sysfs-gpio/compare/0.5.1...0.5.2`
```
Comment (tpmanley) [inline CHANGELOG.md]: I'll remove those extra backtics

**Comment by tpmanley** [comment]
@nastevens or @posborne can you give this a final look at merge if it's OK? Adam has been testing with this.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [0.5.2] - 2018-03-02
```
Comment (nastevens) [inline CHANGELOG.md]: I think the plan is to release 0.5.2 right after this PR is merged (which is fine). But I'd still like the version roll to be a separate commit instead of being squashed into the `active_low` change.

Also, there doesn't seem to be an "unreleased" section for this changelog, could you please add one?

Comment (tpmanley) [reply CHANGELOG.md]: @nastevens , sure thing, pushed the fix

**Comment by posborne** [comment]
There's CI problems but those can be fixed separately.  Passing on stable on Linux and the OSX failure seems superficial (we just have a stub for OSX).
